lib/src/tag/_specs.dart

```diff
@@ -90,6 +91,12 @@ class Runtime {
     tag: PanaTags.runtimeWeb,
   );
 
+  static final wasm = Runtime(
+    'wasm',
+    {..._onAllPlatforms, 'ui', 'ui_web', 'js_interop', 'js_interop_unsafe'},
```

Comment:
BUT!!! 

We need to check the EXCLUSION of `dart:js`, `dart:js_util`, `dart:html` (and) friends, and `pkg;js`!!

Reply:
This is an allow-list: if a package imports/exports anything else outside of these (e.g. `dart:js`), it won't get the tag. IIRC `package:js` exports `dart:js_util` and if something depends on it, it will be applied to it too.

Reply:
Do we test that?

Reply:
I'd love to have a test that validates each of these independently. just to be sure!

Reply:
Not specifically these, but 3 packages in our end2end tests got the tag (`async`, `http`, `lints`), while others e.g. `dnd` and `url_launcher` did not get it, which seems to be about right to me.

Reply:
Added some tests - they revealed problems when the package is a flutter package it would never get the tag because we were special-casing for the dart sdk. I tried to generalize.